src/v8/isolate_scope.rs

```diff
@@ -82,17 +116,24 @@ impl<'isolate> V8IsolateScope<'isolate> {
     pub(crate) fn new_dummy(isolate: &'isolate V8Isolate) -> V8IsolateScope<'isolate> {
         V8IsolateScope {
             isolate,
-            inner_handlers_scope: std::ptr::null_mut(),
-            inner_isolate_scope: std::ptr::null_mut(),
+            _storage: None,
         }
     }
 
     /// Creating a new context for JS code invocation.
-    #[must_use]
     pub fn new_context(&self, globals: Option<&V8LocalObjectTemplate>) -> V8Context {
         V8Context::new(self.isolate, globals)
     }
 
+    /// Returns a [V8ContextScope] if it has already been entered and
+    /// created for this isolate and isolate scope.
+    pub fn get_current_context_scope<'isolate_scope>(
+        &'isolate_scope self,
+    ) -> Option<V8ContextScope<'isolate_scope, 'isolate>> {
+        V8Context::get_current_raw_ref_for_isolate(self.isolate)
+            .map(|p| V8ContextScope::new_for_ref(p, true, self))
```

Comment:
I believe that it should get a `false` here right? We probably do not want to exit the context when this object drop?